change/office-ui-fabric-react-2021-07-19-00-42-30-users-tahsia-fixMessageBarStyling.json

```diff
@@ -0,0 +1,9 @@
+{
+  "type": "patch",
```

Comment:
This change file doesn't look right--it should be for `@fluentui/react` not `office-ui-fabric-react`, and some of the fields shouldn't be there. What command did you use to generate it?